### PR TITLE
Initial plugin

### PR DESCRIPTION
**Pull request by valentinnodan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# mdx-intellij-plugin
\ No newline at end of file
+# mdx-intellij-plugin
+
+[MDX](https://mdxjs.com/) is an authorable format that lets you seamlessly write JSX in your Markdown documents.You can import components, such as interactive charts or alerts, and embed them within your content.This makes writing long-form content with components a blast
```
Comment (anstarovoyt) [inline README.md:3]: actually the description must be about the plugin, not about the mdx

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+*.class
```
Comment (anstarovoyt) [inline .gitignore:1]: also
```
.idea
```

File: build.gradle [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    testCompile group: 'junit', name: 'junit', version: '4.12'
+}
+
+// See https://github.com/JetBrains/gradle-intellij-plugin/
+intellij {
+    version = "IU-2020.1.2"
+}
+patchPluginXml {
+    changeNotes """
+      Add change notes here.<br>
```
Comment (anstarovoyt) [inline build.gradle:23]: it is better to keep the notes empty instead of using the "template" txt

File: src/main/resources/META-INF/plugin.xml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<idea-plugin>
+    <id>org.example.mdx-intellij-plugin</id>
+    <name>MDX</name>
+    <vendor>JetBrains</vendor>
+
+    <description><![CDATA[
+    MDX is an authorable format that lets you seamlessly write
```
Comment (anstarovoyt) [inline src/main/resources/META-INF/plugin.xml:7]: the same thing: it is about the plugin, not about the technology

File: src/main/resources/META-INF/plugin.xml [hunk 5]
```diff
@@ -0,0 +1,24 @@
+<idea-plugin>
+    <id>org.example.mdx-intellij-plugin</id>
```
Comment (anstarovoyt) [inline src/main/resources/META-INF/plugin.xml:2]: mdx.js

Comment (valentinnodan) [reply src/main/resources/META-INF/plugin.xml]: Changed id to mdx.js, but I am not sure that I understood you right

**Comment by valentinnodan** [comment]
Wrote an abstract information about plugin, will improve it in the process of development.

**Pull request merged**